operator/src/test/java/oracle/kubernetes/operator/helpers/CrdHelperTest.java

```diff
@@ -242,8 +241,7 @@ public void whenReplaceFails_scheduleRetry() {
     Step scriptCrdStep = CrdHelper.createDomainCrdStep(KUBERNETES_VERSION_16, null);
     testSupport.runSteps(scriptCrdStep);
 
-    testSupport.verifyCompletionThrowable(FailureStatusSourceException.class);
```

Comment:
As above, the test no longer does what its name says it does.

Reply:
I have changed the name of test method to reflect current purpose and also added back one of the previous assertion. The only change is to assert that CRD replace failure message is logged instead of verifying throwable.